docs/getting-started/installation.md

```diff
@@ -248,6 +251,44 @@ podAnnotations: {}
 
 > **Tip**: List all releases using `helm list`.
 
+## Docker Desktop Extension
+
+Install the Trivy Docker Desktop Extension through the Extension Marketplace in Docker Desktop:
+<img src="docs/imgs/docker-desktop.png">
+
+Once installed, you can 
+
+- Scan public and private images that are either locally or in the Docker Hub.
+- Generate an SBOM for your container images.
+
+Watch the [tutorial for further information.](https://youtu.be/99tiVZLMujs)
```

Comment:
As for `Trivy Operator Installation` and `Docker Desktop Extension`, can we just put "Other tools" and say `See the `Ecosystem` tab?

Reply:
Done